docs/grafana.rst

```diff
@@ -0,0 +1,320 @@
+`Grafana <https://grafana.com/>`_ is the open source analytics and monitoring solution and may be used to view various BEAM metrics in real time during BEAM execution.
+
+Grafana is a tool to display data from data source. In our case the data source is `InfluxDB <https://www.influxdata.com/products/influxdb-overview/>`_ .
+
+
+Docker installation and configuration
+--------------------------------------
+
+There is a docker image with configured Grafana and InfluxDB to work with BEAM thus one need to install docker in order to use Grafana.
+
+Ubuntu Linux users
+^^^^^^^^^^^^^^^^^^^
+
+Requirements
+"""""""""""""""""""
+
+* 64-bit version of one of these Ubuntu versions: Disco 19.04, Cosmic 18.10, Bionic 18.04 (LTS), Xenial 16.04 (LTS)
+
+Installation
+"""""""""""""""""""
+
+There is an official `installation guide for Ubuntu Linux users <https://docs.docker.com/install/linux/docker-ce/ubuntu/>`_ and
+a `Post-installation guide for Linux <https://docs.docker.com/install/linux/linux-postinstall/>`_ with optional procedures
+for configuring Linux hosts to work better with Docker for non-root users.
+
+Otherwise use those commands::
+
+    sudo apt-get update
+    sudo apt-get install docker-ce docker-ce-cli containerd.io docker-compose
+
+
+Mac users
+^^^^^^^^^^^^
+
+Requirements
+"""""""""""""""""""
+
+* Mac hardware must be a 2010 or newer model
+* macOS must be version 10.13 or newer
+* VirtualBox prior to version 4.3.30 must not be installed as it is not compatible with Docker Desktop.
+
+
+Installation
+"""""""""""""""""""
+
+There is an official `installation guide for Mac users <https://docs.docker.com/docker-for-windows/install>`_.
+
+Otherwise download and install `Docker CE for Mac (Stable) <https://download.docker.com/mac/stable/Docker.dmg>`_.
+
+
+Windows users
+^^^^^^^^^^^^^^^^^
+
+Requirements
+"""""""""""""""""""
+
+* Windows 10 64-bit (Pro, Enterprise, or Education (Build 15063 or later)) - these are pre-req for Docker Desktop.
+* Enable Hyper-V & Containers Windows features (usually done by docker installer).
+* BIOS-level hardware virtualization support must be enabled in the BIOS settings (here’s a `guide <#virtualization>`_ on how to do it)
+
+Installation
+"""""""""""""""""""
+
+There is an official `installation guide for Windows users <https://docs.docker.com/docker-for-windows/install>`_.
+
+Otherwise download and install `Docker CE for Windows (Stable) <https://download.docker.com/win/stable/Docker%20for%20Windows%20Installer.exe>`_.
+
+Virtualization
+""""""""""""""""""
+
+Windows users should also check if virtualization is enabled in firmware. To do so, check the Performance tab on the Task Manager:
+
+.. figure:: _static/figs/grafana-docker-windows-virtualization.png
+
+    check if virtualization is enabled
+
+If not, virtualization should be enabled in BIOS:
+
+* Reboot your computer
+*
+
+    Right when the computer is coming up from the black screen, press Delete, Esc, F1, F2, or F4. Each computer manufacturer uses
```

Comment:
Why is this on a separate line from the bullet?

Reply:
This is how sphinx parse things: if it is on a same line, then all text should be on one line, but I wanted to split text into few lines for better readability during writing, so, I put these lines as a separate paragraph.

This is how it loosk after processing:

![image](https://user-images.githubusercontent.com/5393513/78023717-d6bcb480-735f-11ea-9d94-a1d4380c1d97.png)

Reply:
Oh, I see, so the spacing is not translated into the end result - then that is fine...although odd. I guess HTML is the same and that's why you have to use explicit line break or nbsp